zookeeper-client/zookeeper-client-c/src/zookeeper.c

```diff
@@ -1457,7 +1457,7 @@ zhandle_t *zookeeper_init_ssl(const char *host, const char *cert, watcher_fn wat
     zcert.ca = strtok(strdup(cert), ",");
     zcert.cert = strtok(NULL, ",");
     zcert.key = strtok(NULL, ",");
-    zcert.passwd = strtok(NULL, ",");       
+    zcert.passwd = strtok(NULL, "");
```

Comment:
I think this need a test. I find no specified behavior for [`strtok(str, "")`](https://cplusplus.com/reference/cstring/strtok/). 

Besides this, I think this change require us to decide whether and how possible more fields are allowed in future.